src/DayColumn.js

```diff
@@ -295,6 +295,13 @@ class DayColumn extends React.Component {
       }
     }
 
+    let selectorClicksHandler = (box, actionType) => {
+      if (!isEvent(findDOMNode(this), box))
+        this._selectSlot({ ...selectionState(box), action: actionType })
+
+      this.setState({ selecting: false })
+    }
```

Comment:
Refactor handler functions, as they are absolutely the same for `click` and `doubleClick` events, except `action` property.